src/utils.js

```diff
@@ -669,6 +670,60 @@ export const parseContentTypeFromResponse = (response) => {
     };
 };
 
+/**
+ *
+ * @param response
+ * @return {undefined|Array}
+ */
+export const getCookiesFromResponse = (response) => {
```

Comment:
Let's put this somewhere else, we're not generally working with response streams and `utils.js` is already quite bloated.